pom.xml

```diff
@@ -14,31 +14,31 @@
   ~  See the License for the specific language governing permissions and
   ~  limitations under the License.
   -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>wfswarm-rest-http-secured</artifactId>
+  <groupId>io.openshift.booster</groupId>
+  <version>7.0.0-redhat-1-SNAPSHOT</version>
```

Comment:
Did you maybe confuse `pom.xml` and `pom-redhat.xml`? This belongs to `pom-redhat.xml`.

Reply:
good catch